content/documents/order-returns.mdx

```diff
@@ -0,0 +1,157 @@
+---
+type:           article
+title:          Introducing Order Returns
+description:    New set of features to enable all aspects of the Order Returns process.
+author:         mposthumus
+publishDate:    2022-05-16    
+tags:           ["Concepts", "Marketplaces", "Orders"]
+---
+
+We’re excited to present a new set of features to enable all aspects of the Order Returns process. These features will be available in all regions and environments by the end of May 2022.
+
+## Key Highlights
+- An `OrderReturn` must be affiliated with an existing `Order`, and `ItemsToReturn` must be affiliated with existing `LineItems`
+- An `OrderReturn` is not required to have any `ItemsToReturn`, and `ItemsToReturn` are not required to have a `Quantity` specified 
+   - This allows flexibility for providing refunds not associated with the return of a physical item
+   - Examples include refunding expedited shipping charge due to processing delays or providing a partial discount on an item that arrived damaged 
+- A user with the `Shopper` role can create an `OrderReturn` for any order from them (`Order.FromUserID`)
+   - Anonymous users are prohibited from creating or modifying `OrderReturns`
+- A user with the `OrderAdmin` role can create an `OrderReturn` for any `Order` visible to them
+   - For marketplace owners this is all orders in the marketplace 
```

Comment:
In some spots, Marketplace Owner is capitalized, and here it is not.  I'm not sure which is correct, just bringing it to light.

Reply:
thanks I corrected it!